.vib/ejbca/cypress/cypress/integration/ejbca_spec.js

```diff
@@ -0,0 +1,39 @@
+/// <reference types="cypress" />
+
+it('allows to enrol and verify certificate', () => {
+  const certFile = `cypress/downloads/${Cypress.env('username')}.p12`;
+  cy.task('fileExists', certFile).then((exists) => {
+    // A user can only enrol once
+    if (!exists) {
+      cy.visit('/ejbca/enrol/keystore.jsp');
+      cy.get('#textfieldusername').type(Cypress.env('username'));
+      cy.get('#textfieldpassword').type(Cypress.env('password'));
+      cy.get('#buttonsubmitusername').click();
+      cy.get('#tokenKeySpec').select('RSA_2048');
+      cy.get('#certprofile').select('ENDUSER');
```

Comment:
I would parametrize these constants, be it by using a fixtures file or by using a `const` variable. IIRC, we usually opt for the fixtures solution.